docs/examples/external-dns.yaml

```diff
@@ -50,6 +50,8 @@ spec:
         app: external-dns
     spec:
       serviceAccountName: external-dns
+      securityContext:
+        fsGroup: 65534
```

Comment:
would you help use `1337` to be consistent with our other examples(in other projects).
technically, any `fsGroups` works :D 

also you need to sign the CLA